CONTRIBUTING.md

```diff
@@ -1,22 +1,37 @@
 # Contribution guide
 
-## How to start?
-
-Contributing is quite easy: suggest ideas and make them done.
-We use [GitHub issues](https://github.com/etna-team/etna/issues) for bug reports and feature requests.
+Let's look at the possible ways to contribute to the project:
+- Voting for suggestions
+  - Vote for the suggestion you are interested in on [GitHub Discussions](https://github.com/etna-team/etna/discussions/categories/improvements)
+- Taking part in discussions
+  - Take part in discussions on [GitHub Discussions](https://github.com/etna-team/etna/discussions)
```

Comment:
May be we need to add more info about tag system in the discussions

Reply:
For now, I think that we'll add tags manually.